website/docs/documents-loading.mdx

```diff
@@ -11,18 +11,13 @@ Any input provided as a source will be recognized by utils automatically.
 
 It also extracts usages of `gql` from code files using [`@graphql-tools/graphql-tag-pluck`](/docs/graphql-tag-pluck).
 
-For notes on typescript, refer to [loaders](/docs/loaders)
-
 ## Usage
 
 ```ts
 const { loadDocuments } = require('@graphql-tools/load')
 const { GraphQLFileLoader } = require('@graphql-tools/graphql-file-loader')
 const { CodeFileLoader } = require('@graphql-tools/code-file-loader')
 
-// load from string
-const document1 = loadDocuments('query { f }')
```

Comment:
Why do we remove this?

Reply:
because it does not work (raises a TypeScript error and runtime error). I assume it is an artifact form the past.

Reply:
Ok I added a test and fixed that part to satisfy TS.